conans/client/new.py

```diff
@@ -111,20 +111,20 @@ class {package_name}TestConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     requires = "{name}/{version}@%s/%s" % (username, channel)
     generators = "cmake"
+    cmake = None
 
     def build(self):
-        cmake = CMake(self.settings)
+        self.cmake = CMake(self.settings)
         # Current dir is "test_package/build/<build_id>" and CMakeLists.txt is in "test_package"
-        cmake.configure(self, source_dir=self.conanfile_directory, build_dir="./")
-        cmake.build(self)
+        self.cmake.configure(self, source_dir=self.conanfile_directory, build_dir="./")
+        self.cmake.build(self)
 
     def imports(self):
         self.copy("*.dll", "bin", "bin")
         self.copy("*.dylib", "bin", "bin")
 
     def test(self):
-        os.chdir("bin")
-        self.run(".%sexample" % os.sep)
+        self.cmake.test(self)
```

Comment:
I like the helper, it is a good contribution, not only for ``test_package`` but for users wanting to launch tests while package building. But I am not sure that using CMake tests by default for the conan-test functionality would be the best, because it is too much coupled to what people understand as unit tests, and these have the meaning of testing the package. So I like the explicit launch of executables in the ``test()`` method. WDYT @lasote?